### DISCUSSION TITLE
TypeScript ts-standart Changing Code Throwing Error

### DISCUSSION THREAD

**Discussion by canerasa93** [standard/eslint-config-standard-with-typescript#1252] [General]
Hi,

i have a code just like this;

```ts
declare module 'redux' {
  interface Dispatch<A extends Action = AnyAction> {
    <S, E, R>(asyncAction: ThunkAction<R, S, E, A>): R
  }
}
```

when i use ts-standart, it changed like this;

```ts
declare module 'redux' {
  type Dispatch<A extends Action = AnyAction> = <S, E, R>(asyncAction: ThunkAction<R, S, E, A>) => R
}
```

and this stuation is throwing error on my app. how can i solve this? can you help us? 
the error is = Duplicate identifier 'Dispatch'

**Comment by mightyiam** [comment]
@canerasa93 would you mind if we close this referring you to [this note](https://typescript-eslint.io/rules/prefer-function-type/#when-not-to-use-it)?
